kong/plugins/zipkin/reporter.lua

```diff
@@ -25,6 +25,26 @@ local span_kind_map = {
 	producer = "PRODUCER";
 	consumer = "CONSUMER";
 }
+
+-- Utility function to convert opentracing tags to zipkin tags
+-- values of zipkin tags should be strings, see [zipkin-api](https://zipkin.io/zipkin-api).
+function to_zipkin_tags(opentracing_tags)
+	zipkin_tags = {}
+	if not opentracing_tags then
+		return zipkin_tags
+	end
+
+	for k, v in pairs(opentracing_tags) do
+		if type(v) ~= "string" then
```

Comment:
Might as well use `tostring` unconditionally: it will return a string unchanged.